profile/profile.go

```diff
@@ -591,6 +592,9 @@ func (l *Location) string() string {
 	if m := l.Mapping; m != nil {
 		locStr = locStr + fmt.Sprintf("M=%d ", m.ID)
 	}
+	if l.IsFolded {
+		locStr = locStr + "(folded) "
```

Comment:
What does the full output look like with the update? One thing I wondered about is whether we should use something like "[F]" instead of "(folded)" to sort of match how the function-level bits are printed out, see code above. But it's hard to say for sure without looking at the output, hence asking.

Reply:
Currently looks like this
```
  2000: 0x2000 M=1 (folded) foo foo.c:2 s=0
             foo_caller foo.c:1 s=0
```

I think [F] would look better, so changing.